scripts/NuGetUtilities.psm1

```diff
@@ -36,6 +36,18 @@ function Get-PackageDirectoryName($configuration) {
     Join-Path $PackageOutputDirectoryRoot $configuration
 }
 
+# Get the value of the MSBuild property that specifies the "package license expression"
```

Comment:
>the [](start = 19, length = 3)

i don't understand the need for all this machinery around a license expression that is not likely to change for this repo.

Reply:
I already had the machinery written and deployed in JSchema.

---
In reply to: [244648353](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648353) [](ancestors = 244648353)

Reply:
right but we don't need it. and now we're carrying out multiple flavors of build scripts across our various repositories. we should prefer keeping things simple and clean or factor things so that we can universally update all our repos with common improvements. 

---
In reply to: [244648592](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648592) [](ancestors = 244648592,244648353)

Reply:
Removed.

---
In reply to: [244648684](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648684) [](ancestors = 244648684,244648592,244648353)